docs/basics/notice.md

```diff
@@ -0,0 +1,14 @@
+<!--
+order: 7
+-->
+
+# Notice
+
+Important information for Etheremint development.
+
+## Cross-Chain Transaction Replay Attack Prevention
+
+Given the fact that Ethermint uses the same transaction signature format as Ethereum for compatibility with Ethereum tooling, and Ethermint's chainIDEpoch corresponds to the chain ID on Ethereum,
+any valid transaction that is able to broadcast to Ethereum is also able to broadcast to Ethermint, and vice versa, which could lead to the Transaction Replay Attack if Ethermint starts the network by using Ethereum reserved chainIDs.
```

Comment:
"any valid transaction that is able to be broadcasted to Ethereum is also able to be broadcasted to Ethermint"

Reply:
-> "... could lead to a transaction replay attack..."

-> "... if Ethermint is initiated using a reserved chainID according to EIP155."